test/functional/eval/system_spec.lua

```diff
@@ -239,6 +254,8 @@ describe('system()', function()
       end
     end)
     it('to backgrounded command does not crash', function()
+      -- cmd.exe doesn't background a command with &
+      if iswin() then return end
       -- This is indeterminate, just exercise the codepath. May get E5677.
       feed_command('call system("echo -n echoed &")')
```

Comment:
So this test now fails?

Reply:
It's passes before and after this change. I disabled but the test isn't doing what it is described to do.